test/lib/dispatcher.test.js

```diff
@@ -59,16 +68,19 @@ describe('Archive Utility', function () {
     });
 
     it('should create output zip file', async function () {
-        sinon.stub(cli, 'log');
+      sinon.stub(cli, 'log');
 
-        await fs.writeFileSync(path.join(tmpDir, 'test.txt'), 'This is my text');
+      await fs.writeFileSync(path.join(tmpDir, 'test.txt'), 'This is my text');
 
-        let {inputPath, inputPathSize} = await dispatcherInputBuild(cli, tmpDir);
-  
-        assert.equal(inputPathSize, 145);
-        assert.ok(await fs.existsSync(inputPath));
-        const zip = new Zip(inputPath, {});
-        assert.ok(zip.getEntry('test.txt') !== null);
+      const { inputPath, inputPathSize } = await dispatcherInputBuild(
+        cli,
+        tmpDir
+      );
+
+      assert(inputPathSize < 150, 'zipped size: expected less than 150 bytes');
```

Comment:
Can we make that between 140 and 150, please? If it drops to 0 something is wrong and we want to know.